webhook/csv.go

```diff
@@ -40,9 +40,14 @@ type ClusterServiceVersionDeploymentScaler struct {
 
 	Decoder           admission.Decoder
 	OperatorNamespace string
+
+	mutex                               sync.Mutex
+	odfOperatorConfigMapResourceVersion string
+	odfOwnedCsvNames                    map[string]bool
```

Comment:
```suggestion
	odfOwnedCsvSet                   map[string]bool
```

Reply:
Here, it makes sense to use the bool so we can return directly without checking the existence of the key.